lib/manager/regex/__fixtures__/Dockerfile

```diff
@@ -243,6 +243,9 @@ COPY --from=tsbuild dist dist
 COPY bin bin
 COPY data data
 
+# openresty/headers-more-nginx-module
+
+ENV NGINX_MODULE_HEADERS_MORE_VERSION=0.30 # github-tags/openresty/headers-more-nginx-module&versioning=loose&extractVersion=^v(?<version>.*)$
```

Comment:
Ok, we need to refactor this tests (can be done later). As those test regex manager won't work in real, because comments on `ENV` are not allowed in dockerfiles

Reply:
![image](https://user-images.githubusercontent.com/1798109/102098308-5378bb80-3e27-11eb-9eb5-d21f1855cc54.png)

Reply:
as for refactoring the regex manager Dockerfile syntax:
we use something like
https://regex101.com/r/KKaMEA/1

Reply:
https://github.com/renovatebot/presets-internal/blob/beca706fd9463b0e4bc614d36d4648e8883029a3/docker-images.json#L25

🙃

Reply:
This refactoring should be done in a separate pr